src/macaron/policy_engine/souffle_code_generator.py

```diff
@@ -21,7 +21,7 @@ class SouffleProgram:
         Set of declaration souffle statements (begin with .decl token)
     directives: None | set[str]
         Set of directives (begin with ".", e.g. .input)
-    rules: None | set[str]
+    rules: None | set[str]-
```

Comment:
I wonder if this is a typo?

Reply:
Fixed https://github.com/oracle-samples/macaron/pull/46/commits/db11e761070bbd8c28841a19da111817ef1432ea